internal/reader/home_chain.go

```diff
@@ -19,7 +19,10 @@ import (
 	"github.com/smartcontractkit/chainlink-ccip/pkg/consts"
 )
 
-//go:generate mockery --name HomeChain --output ./mocks/ --case underscore
+const (
+	PageSize = uint64(500)
```

Comment:
no need to export, also better to rename `defaultConfigPageSize`.
Isn't 500 too much?, I would expect something closer to `10`

Reply:
u[pdated